### PR TITLE
[MLDB-1695] add a test to make sure the order by in custom folds in c…

### PR DESCRIPTION
**Pull request by simlmx**
…lassifier.experiment is honored

### CODE REVIEW

File: testing/MLDB-878_experiment_proc.py [hunk 1]
```diff
@@ -422,6 +421,61 @@ def test_limitoffset(self):
...
+                'rowName': str(no),
+                'columns': [['order', random(), 0]] + [['x' + str(i), x, 0]
+                                              for i,x in enumerate(row)] \
+                           + [['label', random() > 1, 0]]
+
+
+            })
+        mldb.post('/v1/datasets/ds_ob/commit')
+
+        # mldb.log(mldb.get('/v1/query', q='select * from ds_ob order by no',
```
Comment (mailletf) [inline testing/MLDB-878_experiment_proc.py]: remove comments

**Comment by nicolaskruchten** [comment]
Please merge master into this branch so we can get an all-clear from the tests :)

**Comment by mailletf** [comment]
+1

**Comment by mailletf** [comment]
@simlmx peux-tu merger master svp dans ta branche et si les tests passent merger la branche?

**Pull request merged**